### PR TITLE
OTA-1153: upgrade-status: communicate control plane completion clearly

### PR DESCRIPTION
**Pull request by hongkailiu**

### CODE REVIEW

**Comment by Davoska** [comment]
/retitle OTA-1153: upgrade-status: communicate control plane completion clearly

So that we get the `jira/valid-reference label` 🙌

File: pkg/cli/admin/upgrade/status/controlplane_test.go [hunk 1]
```diff
@@ -230,15 +230,15 @@ func TestAssessControlPlaneStatus_Operators(t *testing.T) {
 					available(configv1.ConditionFalse).
 					progressing(configv1.ConditionTrue).operator,
 			},
-			expected: operators{Total: 2, Unavailable: 1},
+			expected: operators{Total: 2, Unavailable: 1, Upgrading: 1, Waiting: 1},
 		},
```
Comment (Davoska) [inline pkg/cli/admin/upgrade/status/controlplane_test.go:234]: Could we add a new test case to check the `Upgraded` field as well?

**Review by Davoska — CHANGES_REQUESTED** [review]
Overall, the PR looks good, @hongkailiu!

Just two minor nitpicks, one open discussion, and a request for a new test case :raised_hands:

File: pkg/cli/admin/upgrade/status/controlplane.go [hunk 2]
```diff
@@ -426,7 +437,7 @@ func (d *controlPlaneStatusDisplayData) Write(f io.Writer) error {
 const controlPlaneStatusTemplateRaw = `= Control Plane =
 Assessment:      {{ .Assessment }}
 Target Version:  {{ .TargetVersion }}
-Completion:      {{ printf "%.0f" .Completion }}%
+Completion:      {{ printf "%.0f" .Completion }}% ({{ .Operators.Upgraded }} operators upgraded, {{ .Operators.Upgrading }} upgrading, {{ .Operators.Waiting }} waiting)
```
[the remark this answers is not in the document]
Comment (hongkailiu) [reply pkg/cli/admin/upgrade/status/controlplane.go]: Done.

File: pkg/cli/admin/upgrade/status/controlplane_test.go (hunk 1 above)
Comment (hongkailiu) [reply pkg/cli/admin/upgrade/status/controlplane_test.go:243]: Done

**Pull request merged**